.github/workflows/close_issues.yml

```diff
@@ -0,0 +1,22 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 365
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 365 days with no activity."
```

Comment:
Could you add to the comment that the issue will close in x days. If it's not explicit, we will forget

Reply:
"This issue is stale because it has been open for 365 days with no activity. This issue will be closed in 14 days if not addressed."

Or something

Reply:
I worded slightly differently before I saw this comment but good suggestion.